ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlans.j2

```diff
@@ -1,16 +1,12 @@
 {# eos - VLANs #}
-{% if vlans is defined and vlans is not none %}
-{%     for vlan in vlans | arista.avd.natural_sort %}
+{% for vlan in vlans | arista.avd.natural_sort %}
 !
 vlan {{ vlan }}
    name {{ vlans[vlan].name }}
```

Comment:
add check for name

Reply:
Fixed with commit d42782a527b53acfb04121f714f34954f98cf806